build/build-treemap.js

```diff
@@ -19,10 +18,15 @@ const {LH_ROOT} = require('../root.js');
  */
 function buildStrings() {
   const locales = require('../lighthouse-core/lib/i18n/locales.js');
-  const UIStrings = require(
-    // Prevent `tsc -p .` from evaluating util.js using core types, it is already typchecked by `tsc -p lighthouse-treemap`.
-    '' + '../lighthouse-treemap/app/src/util.js'
-  ).UIStrings;
+  // TODO(esmodules): use simple import when build/ is esm.
+  // const UIStrings = require(
```

Comment:
remove this?

Reply:
I think the comment L23 will need to be revived, but it might be forgotten and cause trouble later without this here. would prefer to leave it. all these `TODO(esmodules)` will be cleaned up eventually.

Reply:
just leave the comment from L23 then? this `require` is never coming back :)